java-driver-parent.ipr

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In general we don't check in internal IntelliJ files.  Both this .ipr file and the .iws file below fall into this category so my suggestion would be to remove these from your PR!